pkg/flow/flow.go

```diff
@@ -202,45 +218,39 @@ func newController(modReg *moduleRegistry, o Options) *Flow {
 			HTTPListenAddr:  o.HTTPListenAddr,
 			DialFunc:        dialFunc,
 			ControllerID:    o.ControllerID,
-			NewModuleController: func(id string) controller.ModuleController {
+			NewModuleController: func(id string, keepServices []string) controller.ModuleController {
```

Comment:
nit: The naming feels a little off for `keepServices`? Should these be `availableServices` or follow `needsServices`?